### PR TITLE
first fork push

### PR DESCRIPTION
**Pull request by satish-chandra1**
program updated

### CODE REVIEW

File: room82Practice/src/test/java/room82Practice/noon/Satishfork.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package room82Practice.noon;
```
Comment (Mrinmoy157) [inline room82Practice/src/test/java/room82Practice/noon/Satishfork.java:1]: add javadoc

File: room82Practice/src/test/java/room82Practice/noon/Satishfork.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package room82Practice.noon;
+
+import org.testng.annotations.Test;
+
+public class Satishfork {
+	@Test
+	public void practicefork() {
+		System.out.println("fork");
```
Comment (Mrinmoy157) [inline room82Practice/src/test/java/room82Practice/noon/Satishfork.java:12]: add another comment

File: room82Practice/src/test/java/room82Practice/noon/Satishfork.java (hunk 1 above)
Comment (satish-chandra1) [reply room82Practice/src/test/java/room82Practice/noon/Satishfork.java:1]: java doc added

File: room82Practice/src/test/java/room82Practice/noon/Satishfork.java (hunk 2 above)
Comment (satish-chandra1) [reply room82Practice/src/test/java/room82Practice/noon/Satishfork.java:12]: added

**Pull request merged**